SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialOutputDevice.h

```diff
@@ -17,13 +17,15 @@ class SPATIALGDK_API FSpatialOutputDevice : public FOutputDevice
 
 	void AddRedirectCategory(const FName& Category);
 	void RemoveRedirectCategory(const FName& Category);
-	void SetVerbosityFilterLevel(ELogVerbosity::Type Verbosity);
+	void SetVerbosityLocalFilterLevel(ELogVerbosity::Type Verbosity);
```

Comment:
Are these APIs ever used?

Reply:
Yes, without them SpatialOutputDevice.cpp has 'not a member' error

Reply:
Right, but these functions aren't actually called from anywhere. I think we might still want to keep them and in fact it would be nice to add the ability to change this verbosity at runtime. Perhaps this could be a follow up ticket?

Reply:
Ticket created!